### DISCUSSION TITLE
log4j vulnerability and Photoprism

### DISCUSSION THREAD

**Discussion by quamok** [photoprism/photoprism#1806] [Q&A] [answered]
Hello, I'd like to know if the new CVE 10 log4j vulnerability affects Photoprism in any way. My guess is it does not, but I prefer to be safe and ask :)

source: https://nvd.nist.gov/vuln/detail/CVE-2021-44228

Thanks!

**Comment by lastzero** [comment, marked as answer]
https://twitter.com/lastzero/status/1469803677942755332

PhotoPrism is based on Go and JavaScript, so at best a backend service that uses Elasticsearch internally may be affected. I didn't want to post this publicly, but you asked for it and I don't want to sleep well at night.

**Reply by lastzero** [reply]
Found a solution:
https://github.com/bradfitz/jndi
